src/pages/login/LoginPage.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Input from '../../shared/components/Input/Input';
+import Button from '../../shared/components/Button/Button';
+import ROUTES from '../../shared/constants/paths/paths';
+import './styles/LoginPage.scss';
+
+export default class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      login: '',
+      password: '',
+      error: '',
+    };
+  }
+
+  onLoginChange = (e) => {
```

Comment:
you can create a single method for handling multiple inputs values
you can pass name to input and use `setState({ [e.target.name]: e.target.value })`